From 46dd5449e8ddc533a15ec35092387d048c2cd017 Mon Sep 17 00:00:00 2001
Message-Id: <46dd5449e8ddc533a15ec35092387d048c2cd017.1413836944.git.chang-joon.lee@intel.com>
In-Reply-To: <5d8aa9994fff43965a6fab8c9a528b8b47a9d624.1413836944.git.chang-joon.lee@intel.com>
References: <5d8aa9994fff43965a6fab8c9a528b8b47a9d624.1413836944.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Thu, 25 Sep 2014 11:32:56 +0100
Subject: [PATCH 70/71] REVERTME [VPG]: drm/i915: Fix compiler warning in
 postpone flip

Fix up the compiler warning introduced by:
commit cbff18d3d8012eddc331e510a5836b00c2ccaea0
Author: ian-lister <ian.lister@intel.com>
Date:   Mon Dec 23 10:26:31 2013 +0000

    REVERTME [VPG]: drm/i915: Timeout Detection Recovery for Engine hangs

For: GMINL-1778
Change-Id: Ifaf42c1481d39d0690a25b40496c8a4a036c366b
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 5003202..8ddad75 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -10009,7 +10009,7 @@ static int intel_postpone_flip(struct drm_i915_gem_object *obj)
 
 	ring = obj->ring;
 
-	dev_priv = ring->dev;
+	dev_priv = to_i915(ring->dev);
 	ret = i915_gem_check_wedge(&dev_priv->gpu_error,
 				   dev_priv->mm.interruptible, obj->ring);
 	if (ret)
-- 
1.7.9.5

